app/src/main/java/com/infomaniak/drive/ui/fileList/multiSelect/MultiSelectFragment.kt

```diff
@@ -203,14 +209,18 @@ abstract class MultiSelectFragment(private val matomoCategory: String) : Fragmen
         val isNotOfflineBulk = type != BulkOperationType.ADD_OFFLINE && type != BulkOperationType.REMOVE_OFFLINE
 
         if (areAllFromTheSameFolder && isNotOfflineBulk && (canBulkAllSelectedFiles || hasEnoughSelectedFilesToBulk)) {
-            sendBulkAction(
-                fileCount, BulkOperation(
-                    action = type,
-                    fileIds = if (multiSelectManager.areAllSelected) null else selectedFiles.map { it.id },
-                    parent = multiSelectManager.currentFolder!!,
-                    destinationFolderId = destinationFolder?.id,
+            with(multiSelectManager) {
+                sendBulkAction(
+                    fileCount, BulkOperation(
+                        action = type,
+                        fileIds = if (areAllSelected) null else selectedFiles.map { it.id },
```

Comment:
use `selectedItemIds` instead of

Reply:
This is not the same list here, we can't use the `selectedItemsIds`.

Reply:
Here, we are using the list computed by the `getValidSelectedItems` method.